scss/components/_summary-tab.scss

```diff
@@ -91,6 +97,10 @@
   flex-grow: 1;
 }
 
+.civicase__summary-tab-tile__inner-title {
+  margin-top: 0;
+}
```

Comment:
both these two changes in this partial seem very arbitrary, what are they for?

Reply:
I'll remove these changes. They were added before using the new grid and now they are not needed. I forgot to remove them.